include/wlr/types/wlr_client_buffer.h

```diff
@@ -13,9 +13,25 @@
 #include <wayland-server-core.h>
 #include <wlr/types/wlr_buffer.h>
 
+struct wlr_renderer;
+struct wlr_client_buffer;
+
+struct wlr_client_buffer_impl {
+	bool (*is_instance)(struct wl_resource *resource);
+	bool (*get_buffer_size)(struct wl_resource *resource,
+			struct wlr_renderer *renderer, int *width, int *height);
```

Comment:
The `struct wlr_renderer *` argument is only required because of `wl_drm`. Would be nice to get rid of it somehow.

Reply:
As soon as https://github.com/swaywm/wlroots/pull/2708 lands